media/static-bundles.json

```diff
@@ -1398,7 +1400,8 @@
         "js/base/mozilla-fxa-init.js",
         "js/base/mozilla-fxa-form.js",
         "js/base/mozilla-monitor-button.js",
-        "js/firefox/accounts-2019.js"
+        "js/firefox/accounts-2019.js",
+        "js/firefox/whatsnew/account-conditional-ctas.js"
```

Comment:
This script is fine for /whatsnew, but I think it may be worth creating a separate piece of code to run on /accounts? The reason is that on /accounts we want to call `Mozilla.FxaForm.init();` even on non-Firefox browsers, and the `getFxaDetails();` function doesn't really need to be called in that scenario.

I'd suggest adding some code to `accounts-2019.js` for this:

```javascript
if (Mozilla.Client.isFirefoxDesktop) {
    Mozilla.Client.getFxaDetails(function(details) {
        if (details.setup) {
            Mozilla.MonitorButton.init();
        } else {
            Mozilla.FxaForm.init();
        }
    });
} else {
    Mozilla.FxaForm.init();
}